packages/node_modules/@ciscospark/redux-module-mercury/src/index.js

```diff
@@ -40,12 +41,18 @@ function updateStatusConnected(value) {
 
 export function connectToMercury(spark) {
   return (dispatch) => {
+    metrics.saveElapsedTime(`mercuryConnection:start`);
+    metrics.sendTempMetric(`mercuryConnection:start`, spark);
     spark.mercury.on(`change:connecting`, () => {
       dispatch(updateStatusConnecting(spark.mercury.connecting));
     });
     spark.mercury.on(`change:connected`, () => {
+      metrics.saveElapsedTime(`mercuryConnection:end`);
+      metrics.sendTempMetric(`mercuryConnection:end`, spark);
+      metrics.sendDurationMetric(`mercuryConnection:duration`, `mercuryConnection:start`, `mercuryConnection:end`, spark);
```

Comment:
Does it makes sense to change these to constants? They are used a few different times and I'm prone to making typing mistakes.